packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -660,7 +656,8 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
   }
 
   private _onColumnResized(resizingColumn: IColumn, newWidth: number) {
-    let newCalculatedWidth = Math.max(resizingColumn.minWidth || MIN_COLUMN_WIDTH, newWidth);
+    const minWidth = Math.min(resizingColumn.minWidth || MIN_COLUMN_WIDTH, MIN_COLUMN_WIDTH);
```

Comment:
Is this right? What if I want a `minWidth` for my column larger than the `MIN_COLUMN_WIDTH` constant?

Reply:
For the same reason that a user should be allowed to manually resize greater than the initially specified maxWidth, they should be able to resize to less than the minWidth if they need to view information in another justified column better.  This does not affect the initial column width calculation.

Reply:
Okay, I understand and agree. Can we clarify that behavior in the props jsdoc documentation?

Reply:
I'm still not sure that this is necessary, and concerned about the impact of this in fixed layout mode.
A column with small `minWidth` (less that 100px) won't get impacted by this change, it can still be resized interactively to its `minWidth`.
A column with a big `minWidth`, on the other hand, now can be shrunk to 100px even when it has `minWidth: 200`, and this is a behavior change, and IMO inconsistent with the behavior of small columns.
I see how this can break expectations from consumers, and I say this is not necessary because if desired, this can be achieved using `minWidth` explicitly with a low value.

Reply:
As also discussed offline, just to be clear, setting a reasonably wide minWidth is required for the initial column rendering to make a best guess for showing its contents for the common case.  But that should not prevent a user from shrinking a column that's not important for them so they have more room for the columns that are.  Simply setting a lower minWidth does not solve that problem as it would then make the initial rendering potentially too narrow for everyone.  We should allow the initial width to be 200px, but allow a user to shrink it down if they so desire.  I just don't see any value in forbidding this, but I would be happy to hear a scenario where it needs to be.  Maybe I'm mistaken.  The ability to resize smaller than minWidth will be particularly useful for a justified layout when the total table width is constant, so we should at least consider allowing it for that.

Reply:
I thought we settled on adding a different layoutmode? One that would use min/max only for the automatic adjustment, and one that would enforce it also for manual resize

Reply:
IMO a new layout mode would add extra complexity. We should just clarify semantic of current properties and add new ones if necessary. So:
- `minWidth` stands for the minimum initial size of the column, *plus* it sets a limit to the minimum size a user can resize it.
- `maxWidth` stands for the maximum initial size of the column in justified layout. It's only used in fixed layout if `minWidth` is not specified, and only as initial size. **It doesn't prevent the user from extending the column width father**

I learned this inconsistency between both properties the hard way. I can think of solutions to this (like Jay's idea of `maxUserWidth` which would bring by hand a `minUserWidth`), but IMO they are beyond this fix.

Reply:
@pablonete That's your interpretation of the props. If I understood @jayspadie correctly, he strongly disagrees with your interpretation of `minWidth`, for example. 

If we want to make everyone happy, we would have to differentiate between 
- `minWidth` and `maxWidth` are used for initial sizing, and
- `minWidth` and `maxWidth` are used for initial sizing **and** also constrain any manual resize

Reply:
Obviously, I'm in favor of the user being able to resize columns as they so choose, particularly with a justified layout where the total width will be constrained.  So, Pablo and I still disagree on limiting the user on ```minWidth```, but at least there seems to be agreement not to limit on ```maxWidth```.  So when it comes to the 2 modes you suggest, I can't see a reason for the second where the user is limited by both min and max width.  I don't recall anyone asking for that, and we didn't have a limit on the max width before.  As I mentioned offline, if we are going to consider 2 modes, then I would vote that one would be a revert to the behavior that we had before where a table initially rendered as justified but would change to fixed sizing, allowing the overall width to expand, when the user widened a column.  The other mode remains justified with a fixed overall width.  Neither mode should impose an arbitrary limit to the user resizing the columns as they wish.  But I'm not necessarily pushing for 2 modes.  I just want to give the user freedom with resizing columns, but I don't want to keep debating it.  Whatever you guys want to do.

Reply:
I personally don't care about two modes. I assumed that Pablo had a reason for this change, so him making the change, for me that's someone "asking" for this mode. You clearly disagree so if you two don't converge on one behavior the only solution would be two modes. Or is that not correct? Maybe I'm misunderstanding your positions.

For the `minLimit`: maybe the control I'm hosting in a column doesn't degrade nicely when there is not enough horizontal space, etc. so I want to prevent users from shrinking the column past that point.

Reply:
I think we all agree on the `maxWidth` change, my only concern was about `minWidth` and the example you mention of a hosted component that doesn't support a small width is a good reason to not change that logic here.

Reply:
The unnecessary ```maxWidth``` resize restriction was the primary issue that lead to the bug and this PR, and we all agree it should be removed, so we should get that part in.

I don't agree with the reasoning for a resize restriction on the initial ```minWidth``` since the user is explicitly not interested in the content at the time (degraded or not), but that is a lesser problem and I'm OK with removing it if you guys insist.

@pablonete , since you made the initial changes to the justified layout and have done a lot of testing recently for the ported fix in our local copy used in VSTS, would you like to either update this PR as needed or simply create a new one with just the fix to remove the ```maxWidth``` restriction?

Reply:
Took offline, we're deferring the `minWidth` change until we agree, and making a smaller PR to fix only the `maxWidth` bug. Please continue here: [PR 2978](https://github.com/OfficeDev/office-ui-fabric-react/pull/2978)
We can safely close this PR.